src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_format.py

```diff
@@ -224,9 +226,29 @@ def _get_value(item, *args):
         return ' '
 
 
-def _get_build_trigger(image_update_trigger, git_source_trigger):
+def _get_timer_triggers(item, *args):
+    """Get timer trigger schedules from a dict.
+    :param dict item: The dict object
+    """
+    try:
+        for arg in args:
+            item = item[arg]
+        if item:
+            schedules = []
+            for trigger in item:
+                schedules.append(trigger['schedule'])
+            return ', '.join(schedules)
+        else:
+            return ' '
```

Comment:
You don't need the else for return here. This will probably throw a lint error

Reply:
It is fine to return empty strings for pylint. See the previous example for `_get_value`

Reply:
No, I mean 
else: 
  return ' '
Should just be the return. There is no need for the else. If it doesn't enter the if then just return. No need to type else

Reply:
Removed this else. Thanks!